BistroQ.Services/Services/OrderService.cs

```diff
@@ -60,10 +61,33 @@ public async Task<DetailOrderDto> GetOrder(int tableId)
 		{
 			throw new ResourceNotFoundException("Order not found");
 		}
-
+		
 		order.OrderItems = _mapper.Map<List<OrderItem>>(order.OrderItems);
 
-		return _mapper.Map<DetailOrderDto>(order);
+		var result = _mapper.Map<DetailOrderDto>(order);
+		
+#region braindead-code
```

Comment:
Add a `GetByIdsAsync` method to `INutritionRepository` can resolve this

Reply:
Done.